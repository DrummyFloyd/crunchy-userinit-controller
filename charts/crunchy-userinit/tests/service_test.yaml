# yaml-language-server: $schema=https://raw.githubusercontent.com/helm-unittest/helm-unittest/main/schema/helm-testsuite.json
suite: Service
templates:
  - service.yaml
release:
  name: test
  namespace: test-ns
chart:
  version: 0.0.0
  appVersion: 0.0.0
set:
  image:
    repository: ghcr.io/drummyfloyd/crunchy-userinit-controller
    tag: 0.0.0
tests:
  - it: should match snapshot with defautl
    asserts:
      - matchSnapshot: {}
  - it: should override spec
    set:
      service:
        type: LoadBalancer
        port: 8888
    asserts:
      - equal:
          path: spec.type
          value: LoadBalancer
      - equal:
          path: spec.ports[0].port
          value: 8888
