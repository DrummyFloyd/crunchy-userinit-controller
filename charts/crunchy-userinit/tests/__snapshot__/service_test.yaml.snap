should match snapshot with defautl:
  1: |
    apiVersion: v1
    kind: Service
    metadata:
      labels:
        app.kubernetes.io/instance: test
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: crunchy-userinit-controller
        app.kubernetes.io/version: 0.0.0
        helm.sh/chart: crunchy-userinit-controller-0.0.0
      name: test-crunchy-userinit-controller
    spec:
      ports:
        - name: http
          port: 80
          protocol: TCP
          targetPort: http
      selector:
        app.kubernetes.io/instance: test
        app.kubernetes.io/name: crunchy-userinit-controller
      type: ClusterIP
